server/src/operators/chunk_operator.rs

```diff
@@ -1332,6 +1332,7 @@ pub fn get_slice_from_vec_string(vec: Vec<String>, index: usize) -> Result<Strin
     }
 }
 
+#[inline(never)]
 pub fn get_stop_words() -> Vec<String> {
```

Comment:
This also is a likely culprit, but we've had this function in the code base for a while so I doubt this will be the root cause.